apps/demos/Demos/Chat/Customization/jQuery/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+  <head>
+    <title>DevExtreme Demo</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0" />
+    <script src="../../../../node_modules/jquery/dist/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../../../../node_modules/devextreme-dist/css/dx.light.css" />
+    <script src="../../../../node_modules/devextreme-dist/js/dx.all.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script src="data.js"></script>
+    <script src="index.js"></script>
+  </head>
+  <body class="dx-viewport">
+    <div class="demo-container">
+      <div class="chat-container">
+        <div id="chat"></div>
+      </div>
+
+      <div class="options">
+        <div class="caption">Options</div>
+
+        <div class="option-group">
+          <div class="option">
+            <div id="show-avatar"></div>
+          </div>
+
+          <div class="option">
+            <div id="show-user-name"></div>
+          </div>
+        </div>
+
+        <div class="option-group">
+          <div class="option">
+            <div id="show-day-headers"></div>
+          </div>
+
+          <div class="option">
+            <span>Day Headers Format:</span>
+            <div id="day-headers-format"></div>
+          </div>
+        </div>
+
+        <div class="option-group">
+          <div class="option">
+            <div id="show-message-timestamp"></div>
+          </div>
+
+          <div class="option">
+            <span>Message Timestamp Format:</span>
+            <div id="message-timestamp-format"></div>
+          </div>
+        </div>
+
+        <div class="option-group">
+          <div class="option">
+            <div class="option">
+              <div id="chat-disabled"></div>
```

Comment:
You wrap it in option container twice

Reply:
https://github.com/DevExpress/DevExtreme/pull/28441/commits/7abeedba5198baaf75acd81fdb9e03e779b24d94